src/confcom/azext_confcom/_params.py

```diff
@@ -230,6 +230,12 @@ def load_arguments(self, _):
             required=False,
             help="Feed for the generated policy fragment",
         )
+        c.argument(
+            "image_target",
+            options_list=("--image-target"),
+            required=False,
+            help="Image target where the generated policy fragment is attached",
```

Comment:
does this flag need to be added to any readme/example? do we have an example on how to use it or what an example input looks like?